README.md

```
@@ -0,0 +1,25 @@
+
+workflowを読み込めば誰でも同じ画像・動画が生成できる、というのがComfyUIの最大の魅力ではありますが、これを自分なりにカスタムしようとしたときは、やはりComfyUIの使い方を知っておく必要があります。
+
+このサイトでは特に、生成AIエンジンではなく、データ処理ソフトウェアとしてのComfyUIの機能を多く紹介しています。
+
+未熟で魅力あふれる生成AIの世界を、ともに学んでいきましょう(´ε｀ )
+
+---
+
+# Comfy with ComfyUI
+
+**https://comfyui.nomadoor.net/**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)